Rakefile

```diff
@@ -3,7 +3,8 @@
 GEMSPEC = File.expand_path('prawn.gemspec', __dir__)
 require 'prawn/dev/tasks'
 
-task default: %i[spec rubocop]
+task default: :spec
+task default: :rubocop if RUBY_ENGINE == 'ruby'
```

Comment:
I don't this is needed any more.

Reply:
It seems wasteful to run RuboCop on TruffleRuby, and there might be issues with RuboCop on TruffleRuby, and those are just not interesting to check in this CI.

Reply:
We don't in CI and locally you have a choice to run Rubocop and RSpec separately.

Reply:
Ah, but the CI does `bundle exec rake spec` now, I see, I'll remove the commit.

Reply:
Yes, it's a recent change.